packages/valory/skills/abstract_round_abci/test_tools/rounds.py

```diff
@@ -139,9 +140,8 @@ def get_dummy_tx_payloads(
 class DummyRound(AbstractRound):
     """Dummy round."""
 
-    round_id = "round_id"
     allowed_tx_type = DummyTxPayload.transaction_type
-    payload_attribute = "value"
+    payload_attribute = get_name(DummyTxPayload.value)
```

Comment:
Just as a side note, it seems we provide the payload multiple times. Implicitly before, and explicitly now. We could refactor this in the future, to take the payload once, and infer these two class vars.

Reply:
Yes that was the idea. That's why I am making all these links apparent so the refactor can be optimally done in the end.